SPECS-EXTENDED/pykickstart/pykickstart.spec

```diff
@@ -0,0 +1,804 @@
+# Disable tests by default because they fail to run inside mock builds
+# at the moment, but can run locally.  To build and run tests, do:
+#     rpmbuild -ba --with runtests pykickstart.spec
+%bcond_with runtests
+Summary:        Python utilities for manipulating kickstart files.
+Name:           pykickstart
+Version:        3.36
+Release:        2%{?dist}
+License:        GPLv2 AND MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://fedoraproject.org/wiki/pykickstart
+# This is a Red Hat maintained package which is specific to
```

Comment:
Probably we should double check if this comment is valid, as we have source0 pointing to github

Reply:
I have modified the link to get the source from GitHub but forgot to remove the comment. I have now deleted it.